db/data/20240726061113_add_organization_id_to_activity_notifications.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class AddOrganizationIdToActivityNotifications < ActiveRecord::Migration[6.1]
+  def up
+    ActiveRecord::Base.transaction do
+      organization = Organization.singleton
+      return unless organization
+
+      ActivityNotification.find_each do |notification|
+        notification.organization_id = organization.id
+        notification.save!
+        Rails.logger.debug '.'
```

Comment:
What's up with the dot? 

In previous times, applications would print a dot to stdout to have some IO and thus get priority by the operating system scheduler. :thinking:

Reply:
I like to see output when I run rake tasks so that I know something is happening hehe. Don't know if I should try to break this habit...